t/Controller/Jobset/type-constraints.t

```diff
@@ -0,0 +1,128 @@
+use strict;
+use Setup;
+use Test2::Compare qw(compare strict_convert);
+my %ctx = test_init();
+
+require Hydra::Schema;
+require Hydra::Model::DB;
+require Hydra::Helper::Nix;
+
+use Test2::V0;
+require Catalyst::Test;
+use HTTP::Request::Common;
+Catalyst::Test->import('Hydra');
```

Comment:
I think you probably don't need to import these.

Reply:
I had copied over the HTTP API test as scaffolding initially :sweat_smile: